lib/CurrentUser.php

```diff
@@ -10,28 +10,22 @@
 use OCP\IRequest;
 use OCP\IUser;
 use OCP\IUserSession;
+use OCP\L10N\IFactory;
 use OCP\Share\Exceptions\ShareNotFound;
 use OCP\Share\IManager;
 use OCP\Share\IShare;
 
 class CurrentUser {
 
-	/** @var string */
-	protected $identifier;
-	/** @var string|null */
-	protected $cloudId;
-	/** @var string|false|null */
-	protected $sessionUser;
+	protected string|null $identifier;
+	protected string|null $cloudId;
+	protected string|false|null $sessionUser;
```

Comment:
```suggestion
	protected ?string $identifier;
	protected ?string $cloudId;
	protected ?string $sessionUser;
```

Can we avoid the triple type for the `$sessionUser` somehow? Haven't checked all returns but it would be nice if we could get rid of the `false`.

Reply:
See other comment, we need the false value. Even for $cloudId